demo/load/full-page.html

```diff
@@ -56,20 +56,20 @@
   properties: {
 
     animationConfig: {
-      type: Object,
-      value: function() {
-        return {
-          'entry': [{
-            name: 'slide-from-top-animation',
-            node: this.$.toolbar
-          }, {
-            animatable: this.$.grid,
-            type: 'entry'
-          }]
-        };
-      }
+      type: Object
     }
+  },
 
+  attached: function() {
+    this.animationConfig = {
```

Comment:
nit: set this only once, if undefined `this.animationConfig = this.animationConfig || {`